python/sdist/setup.py

```diff
@@ -209,7 +209,7 @@ def main():
         setup_requires=['setuptools>=40.6.3'],
         python_requires='>=3.6',
         extras_require={
-            'petab': ['petab==0.1.7'],
+            'petab': ['petab==0.1.11'],
```

Comment:
```suggestion
            'petab': ['petab>=0.1.11'],
```

Would only set a minimum version requirement (assuming no major backwards incompatibilities in the near future...)